modules/oadp-backing-up-pvs-csi.adoc

```diff
@@ -0,0 +1,95 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/backing_up_and_restoring/backing-up-applications.adoc
+
+[id="oadp-backing-up-pvs-csi_{context}"]
+= Backing up persistent volumes with Container Snapshot Interface snapshots
+
+You back up persistent volumes with Container Snapshot Interface (CSI) snapshots by creating a `VolumeSnapshotClass` CR.
+
+.Prerequisites
+
+* The cloud provider must support CSI snapshots.
+* You must install the OpenShift API for Data Protection (OADP) Operator and the Data Protection Application.
+* You must add the `csi` default plugin and the `EnableCSI` feature flag to the `DataProtectionApplication` manifest:
++
+.Example DataProtectionApplication manifest with CSI
++
+[source,yaml]
+----
+apiVersion: oadp.openshift.io/v1alpha1
+kind: DataProtectionApplication
+...
+spec:
+  configuration:
+    velero:
+      defaultPlugins:
+      - openshift
+      - csi
+    featureFlags:
+    - EnableCSI
+----
+
+.Procedure
+
+. Create a `VolumeSnapshotClass` manifest for CSI, as in the following example:
++
+.Example VolumeSnapshotClass manifest
+[source,yaml]
+----
+apiVersion: snapshot.storage.k8s.io/v1
+kind: VolumeSnapshotClass
+metadata:
+  name: <csi-aws-example>
+  labels:
+    velero.io/csi-volumesnapshot-class: "true"
+driver: <csi_driver> <1>
+deletionPolicy: Retain
+----
+<1> Specify the CSI driver, for example, `ebs.csi.aws.com` for Amazon Web Services EBS.
+
+. Retrieve the `backupStorageLocations` CRs:
++
+[source,terminal]
+----
+$ oc get backupStorageLocations
+----
++
+.Example output
++
+[source,terminal]
+----
+NAME              PHASE       LAST VALIDATED   AGE   DEFAULT
+velero-sample-1   Available   11s              31m
+----
+
+. Create a `Backup` CR, as in the following example:
++
```

Comment:
If you did the previous section, Creating a Backup CR, you already have a `Backup` CR named '<backup>`. I should think that you will not be allowed to do this here. Should the user *edit* the `Backup` CR? 

Also, is there a reason for using the brackets in the name? Seems unusual.

Reply:
You mean <name>? It's a user-editable value.

Reply:
I could have left out the steps for getting the backupStorageLocations and creating a Backup CR. However, we can't put xrefs in modules and it was only a couple lines.

Maybe there is a way to remove the repetition. I'll try to reorganize it a little.